thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/notification/commons/ThirdEyeJiraClient.java

```diff
@@ -61,30 +61,36 @@ private JiraRestClient createJiraRestClient(JiraConfiguration jiraAdminConfig) {
         jiraAdminConfig.getJiraPassword());
   }
 
-  /**
-   * Search for all the existing jira tickets based on the filters
-   */
-  public List<Issue> getIssues(String project, List<String> labels, String reporter, long lookBackMillis) {
-    List<Issue> issues = new ArrayList<>();
-
+  private String buildQueryOnCreatedBy(long lookBackMillis) {
     long lookBackDays = TimeUnit.MILLISECONDS.toDays(lookBackMillis);
     String createdByQuery = "created";
-    if (lookBackDays <= 0) {
+    if (lookBackDays < 0) {
```

Comment:
Can `lookbackDays` be negative? In what case it is negative?

Reply:
lookBackDays is the number of days beyond which you shouldn't merger (reopen and comment) on jira tickets.
```
=0 - means look back for 0 days which essentially means do not look back. Keeping creating new jira tickets and do not merge.
>0 - means look back for specific number of days.
<0 - Any negative value is used to denote always merge (infinite lookback). This is the default setting.
```